pkgs/development/python-modules/asyncssh/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, buildPythonPackage, fetchPypi, pythonOlder, isPy34, isPy35, isPy36
+, cryptography
+, bcrypt, gssapi, libnacl, libsodium, nettle, pyopenssl }:
+
+buildPythonPackage rec {
+  pname = "asyncssh";
+  version = "1.12.2";
+  disabled = pythonOlder "3.4";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "154zgl527dach974w8fad0aw326rnhxsrx79svad8vz7wn4dycv9";
+  };
+
+  propagatedBuildInputs = [ 
+    bcrypt
+    cryptography
+    gssapi
+    libnacl
+    libsodium
+    nettle
+    pyopenssl
+  ];
+
+  patches = [ ./ASYNCSSH-1.12.2_sspi-stub.patch ];
```

Comment:
I'd prefer deleting the file using `rm` in `postPatch`. And please add a comment why it is deleted.

Reply:
Done